contrib/devtools/README.md

```diff
@@ -40,6 +40,12 @@ would be changed to:
 
 ```// Copyright (c) 2009-2015 The Bitcoin Core developers```
 
+gen-manpages.sh
+===============
+
+A small script to automatically create manpages in ../../doc/man by running the release binaries with the -help option.
+This requires help2man which can be found at: https://www.gnu.org/software/help2man/
```

Comment:
Should probably mention the relevant binaries need to be in PATH, adjust it so the BITCOIND and/or BITCOINCLI environment variables are used, and/or just use the binaries in the build dir.